homework/index.js

```diff
@@ -0,0 +1,28 @@
+$(document).ready(function(){
+
+	$('button').click(function(){
+
+		var data = {
+			'user': $('input').val(),
+			'message': $('textarea').val()
+		};
+
+		//data = JSON.stringify(data);
+
+		$.post('https://fewd20.herokuapp.com/chat', data, function(data, status){
+
+			$('#list').append('<li> name: ' + $('input').val() + ' message :' + $('textarea').val() + '</li>');
```

Comment:
You've just posted the message the user typed.  This should probably be done before the post.